student-store-ui/src/components/Reducer/CheckoutCartReducer.jsx

```diff
@@ -0,0 +1,58 @@
+export default function CheckoutCartReducer(checkoutCart, action){
+    switch (action.type){
```

Comment:
Nice approach with switch case! I recommend checking out the exemplar code to learn about another approach to get exposure